modules/fame_azure-vpn/conf/02-tunnel-status.yaml

```diff
@@ -0,0 +1,16 @@
+module: Azure VPN
+name: IPSEC Tunnel Status
+id: tunnelstatus
+
+transformation: ".fill()"
```

Comment:
It seems to me that `fill` without value is useless.
Also, use extrapolation instead.